pkg/server/preprocessor/preprocessor.go

```diff
@@ -50,6 +50,7 @@ func isTokenStillValid(token router.AccessToken, authInfo skydb.AuthInfo) bool {
 }
 
 func (p InjectUserIfPresent) Preprocess(payload *router.Payload, response *router.Response) int {
+	// TODO: Inject both AuthInfo and user Record
```

Comment:
The implementation of injecting user record looks pretty complicated. Since the preprocessor of injecting AuthInfo would create the auth record if user has master key, I assume that of injecting user would also do the same.

Either the preprocessor call recordSaveHandler, or it call db save directly, and I think both would not be finished in a few lines. Currently only the MeHandler need this and I think the current PR is big enough, so I want to leave it later.